manifests/build.pp

```diff
@@ -146,6 +152,19 @@
     refreshonly => true,
   }
 
+  # In case the rubygems version is set, it should be called before installing bundler. Otherwise you could run into
+  # a series of issues like
+  # https://bundler.io/blog/2019/05/14/solutions-for-cant-find-gem-bundler-with-executable-bundle.html.
+  if $rubygems_version {
```

Comment:
Perhaps it would be a good idea to always update rubygems at install time (eg: remove the `if` and set the default value to empty string). Then users could specify `rubygems_version` if the default (latest) is not suitable. What do you think?

Reply:
I like the idea, just two things which come to my mind about this:

* The default bundler version could change depending on the rubygems version
* You might encounter new deprecation warnings or breaking changes in rubygems itself

So more generally spoken, this would introduce different behaviors on servers depending on the time of installation. Which might be unexpected. 

For me both options are okey, as it is possible to freeze the versions. Just let me know which one you prefer for `puppet-rbenv`.

Reply:
I prefer to minimize surprises. 😄 Let's leave it the way you wrote it.